docs/reference/known_limitations.md

```diff
@@ -15,25 +15,35 @@ We are developing a new feature that will allow you to modify these settings dir
 
 ## Backups
 
-The limitations related to backups in Percona Everest are as follows:
+Let's delve into the limitations of on-demand backups in Percona Everest. 
+
+### PostgreSQL limitations for on-demand backups
 
 - When attempting to delete a PostgreSQL database that contains backups created with Everest versions older than 1.0.0, the database may become stuck in the **Deleting** state. 
 
     **Workaround**: To prevent this, manually delete any backups created with versions prior to 1.0.0 by using the **Delete** action on the **Backups** page before deleting the database.
 
-- You cannot change the bucket name for a specific backup storage. Doing so will make any backups taken for that bucket unusable.
+- You cannot change the bucket name and region for a specific backup storage. Doing so will make any backups taken for that bucket unusable.
+
+- If you create more than three backup storages, you can use any of the existing storages across on-demand backups and schedules, as long as the total number of storages in use (by existing on-demand backups and schedules) does not exceed three.
```

Comment:
This first part seems redundant to me, what do you this of the following suggestion?
```suggestion
- You can use any of the existing backup storages across on-demand backups and schedules, as long as the total number of storages in use (by existing on-demand backups and schedules) does not exceed three.
```